src/remote_adapter/remote_adapter_client/qt_impl/lua_lib/sdl_remote_adapter_qt_client_lua_wrapper.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+extern "C" {
+#include <lua.h>
+#include <lualib.h>
+#include <lauxlib.h>
+}
+
+namespace lua_lib {
+
+struct SDLRemoteTestAdapterLuaWrapper {
+  static int create_SDLRemoteTestAdapter(lua_State* L);
```

Comment:
@aderiabin functions require doxygen description a d splitting to public/private